client/my-sites/plugins/plugin-details-CTA/usps.tsx

```diff
@@ -145,21 +148,37 @@ export const PlanUSPS: React.FC< Props > = ( {
 	switch ( requiredPlan ) {
 		case PLAN_PERSONAL:
 		case PLAN_PERSONAL_MONTHLY:
-			planText = translate( 'Included in the Personal plan (%(cost)s/%(periodicity)s):', {
-				args: {
-					cost: planDisplayCost as string,
-					periodicity: periodicityLabel,
-				},
-			} );
+			planText = isEnglishLocale
```

Comment:
Ah, good point. I'll add it here and across other PRs as well later. 

To clarify, I think we need to revisit and clean up the conditional expression here with translation checks or not once we have the translations. Leaving them will mean everytime this string is rendered it will have to go through an extra string search, which is not good :)

Reply:
Added in https://github.com/Automattic/wp-calypso/pull/85006/commits/125ccb54af30e109ac245039b0995a9c8641827c